doc/source/management/innodb_expanded_fast_index_creation.rst

```diff
@@ -55,8 +62,10 @@ There’s also a number of cases when this optimization is not applicable:
 Version Specific Information
 ============================
 
-  * :rn:`5.7.10-1`
-    Feature ported from |Percona Server| 5.6
+.. TODO:: Change RC specific information
```

Comment:
Consider doing the same as for column compression with dictonaries, same feature status

Reply:
Added the Availability block at the top which informs that the feature is going to be available in a future release